controller/novels.js

```diff
@@ -6,6 +6,17 @@ const getAllNovels = async (request, response) => {
   return response.send(novels)
 }
 
+const getNovelsById = async (request, response) => {
+  const { id } = request.params
+  const novels = await models.Novels.findOne({
+    where: { id },
+    include: [{ model: models.Authors }, { model: models.Genres }]
```

Comment:
At some point you removed this line and you needed it for the last call to work in your README include: [{ model: models.Authors }, { model: models.Genres }]